src/modules/main.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable  */
```

Comment:
# Optional
- You have disabled the entire linters of the main file. Now nothing shows if your code has to be trusted. Also, Recruiters will not believe that you work with clean codes and without baggs. Please start learning how to fix linters manually, it will help you a lot.